lua/bufferline/nvim.lua

```diff
@@ -1,18 +0,0 @@
---
```

Comment:
I thought this was a clever use of metatables (one of my favorite Lua features) but I think `local foo = vim.api.nvim_foo` is probably good enough, and doesn't incur the cost of string concatenation for function calls.

Reply:
Yes. I've always been annoyed that the neovim API uses very bad namespacing but it's better to remove.